server/src/test/java/io/deephaven/server/session/SessionStateTest.java

```diff
@@ -81,7 +79,7 @@ public void teardown() {
     public void testDestroyOnExportRelease() {
         final CountingLivenessReferent export = new CountingLivenessReferent();
         final SessionState.ExportObject<Object> exportObj;
-        try (final SafeCloseable scope = LivenessScopeStack.open()) {
+        try (final SafeCloseable ignored = LivenessScopeStack.open()) {
```

Comment:
BTW, FYI, no action required, some PL people use the variable _ (that is, single underscore) to mean "dummy" or "ignored"

Reply:
That's a neat fact. I'm not sure if that existed before Scala, but `_` is a very heavy scala ism. You can even use it as an unlabeled single parameter into a lambda (i.e. `_.convertToInt(radix)`). `_` is an invalid identifier in java, so we can't do the same. IntelliJ auto suppresses unused warnings when the name is `ignored`.

Reply:
Ah, didn't realize _ was illegal in Java, thanks. I think I picked up the convention from Go and Rust (where _ is varying degrees of special).

Reply:
The reason that `_` is illegal is that this is coming in a future release, they wanted to reserve it to not break existing code.

https://mail.openjdk.org/pipermail/amber-spec-experts/2022-September/003485.html

As with other languages, you will be able to mark more than one local as `_`, and it looks like even type params will be able to be `_` (as opposed to `?`).